.github/workflows/publish-release.yml

```
@@ -49,10 +49,10 @@ jobs:
           key: ${{ github.sha }}
       - name: Dry Run Publish
         # omit npm-token token to perform dry run publish
-        uses: MetaMask/action-npm-publish@v3
+        uses: MetaMask/action-npm-publish@v4
         with:
           slack-webhook-url: ${{ secrets.SLACK_WEBHOOK_URL }}
-          target-name: metamask-npm-publishers
+          subteam: S042S7RE4AE
```

Nit: A comment explaining what the id is would be nice

```suggestion
          subteam: S042S7RE4AE # @metamask-npm-publishers
```